mxcubecore/HardwareObjects/ICATLIMS.py

```diff
@@ -1112,6 +1112,11 @@ def finalize_data_collection(self, collection_parameters):
                 collection_parameters.get("blSampleId")
             )
 
+            try:
+                metadata["lims"] = HWR.beamline.lims.get_active_lims()
+            except Exception as e:
+                logger.exception("Failed to read get_active_lims.%s")
```

Comment:
```suggestion
                logger.exception("Failed to read get_active_lims.")
```

No need to keep the `%s`.

Reply:
I thinking linting with fail because of the extra `e` in `except Exception as e:`  @antolinos could you run `ruff` please ?